src/app/login/login-page/login-page.component.html

```diff
@@ -1,2 +1,26 @@
-<p>login-page works!</p>
-<button (click)="login()">login</button>
+<form [formGroup]="loginForm" (ngSubmit)="onSubmit()" novalidate data-role="login-form">
+	<div class="col-lg-6">
+		<bb-header-ui headingType="h1" heading="Welcome to golden sample application"></bb-header-ui>
+		<bb-input-validation-message-ui
+			[showErrors]="loginForm.controls.login.invalid && loginForm.controls.login.touched"
+			data-role="login-validation-message"
+		>
+            <span i18n="login is required and minimu leanth is 4 characters validation error">
```

Comment:
ui standards that I have seen in capabilities, usually put the error message below the input

Reply:
i have checked the examples in ui-ang and you are right, it is just my inner feel of how it is better.
fixed